### ISSUE TITLE
Zebra table

### ISSUE THREAD

**Issue by vaclavvanik** [KeeYoMEE/phapko#1]
Nechť má každé druhé tr class="bold"

**Comment by KeeYoMEE** [comment]
Vyřešeno

**Comment by vaclavvanik** [comment]
pře foreachem si nakleduruji `<?php $i = 0; ?>`

ve foreachi pak `<tr<?php echo (++$i % 2) ? ' class="strong"': ''; ?>>`

pripadne s opacnou podminkou, ted nevim, jestli to odpovida zadani.
podivej do manualu na ternary operator a pre/post increment operator

uprav to tedy podle toho, co jsem psal.

* http://www.php.net/manual/en/language.operators.comparison.php#language.operators.comparison.ternary
* http://php.vrana.cz/ternarni-operator.php
* http://php.net/manual/en/language.operators.increment.php
* http://stackoverflow.com/questions/6400518/pre-incrementation-vs-post-incrementation#answers-header

**Comment by vaclavvanik** [comment]
jinac na stylovani se pouziva dost casto css framework twitter bootstrap http://getbootstrap.com/

jsou tam vyreseny layouty (gridy), tabulky, formulare, buttony, navigace atd

**Comment by KeeYoMEE** [comment]
Takže tím spůsobem přes  tr:nth-child(odd)  to nejde?

**Comment by vaclavvanik** [comment]
jo, samozřejmě jde, já ale chtěl, abys to dal přes PHP. jinak další možnosti jsou samozřejmě CSS nebo javascript

**Comment by KeeYoMEE** [comment]
Ok
